packages/apt/preinst.sh.in

```diff
@@ -0,0 +1,18 @@
+#!@TERMUX_PREFIX@/bin/sh
+
+if [ "$1" = "install" ] && [ "$#" -ge 2 ]; then
+	if [ -f "@TERMUX_PREFIX@/etc/apt/@LEGACY_SOURCES@" ]; then
+		echo
+		echo "================================"
+		echo "Legacy repo file found for '@REPO@' repository."
+		echo "The legacy '@LEGACY_SOURCES@' entry will be migrated"
+		echo "to 'sources.list.d/@REPO@.sources' as part of the update."
+		cp -vaf \
+			"@TERMUX_PREFIX@/etc/apt/@LEGACY_SOURCES@" \
+			"@TERMUX_PREFIX@/etc/apt/@LEGACY_SOURCES@.dpkg-old"
+		echo "A backup copy of the old @LEGACY_SOURCES@ repo file"
+		echo "has been saved to @TERMUX_PREFIX@/etc/apt/@LEGACY_SOURCES@.dpkg-old"
```

Comment:
Single quote `@TERMUX_PREFIX@...` and `@LEGACY_SOURCES@` above it. And it should be `apt '@LEGACY_SOURCES@' file` instead of `repo file` here and first line.

Reply:
>I'm pretty sure that's correct as is

Yeah, can't use `@LEGACY_SOURCES@`. Maybe `apt sources config file` instead of `repo file`.

- https://wiki.debian.org/SourcesList#Using__a_text_editor

Reply:
> Yeah, can't use `@LEGACY_SOURCES@`. Maybe `apt sources config file` instead of `repo file`.
> 
> - https://wiki.debian.org/SourcesList#Using__a_text_editor

If there's established upstream terminology we should probably follow it.